packages/core/src/dialects/abstract/query-interface.d.ts

```diff
@@ -218,6 +218,7 @@ export interface AddForeignKeyConstraintOptions extends BaseConstraintOptions {
   references?: {
     table: TableName,
     field: string,
+    fields: string[],
```

Comment:
Also I think this type should require field xor fields, but not both at the same time

Reply:
Done